packages/api/cms-api/src/dam/files/folders.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Get, NotFoundException, Param, Res } from "@nestjs/common";
+import { Response } from "express";
+
+import { FoldersService } from "./folders.service";
+
+@Controller("dam/folders")
+export class FoldersController {
+    constructor(private readonly foldersService: FoldersService) {}
+
+    @Get("/:folderId/zip")
+    async createZip(@Param("folderId") folderId: string, @Res() res: Response): Promise<void> {
```

Comment:
We need to verify that the user can access the scope where the folder is in

Reply:
How would I do that?

Reply:
1. Get the current user
2. Get the scope of the folder (if DAM has enabled scoping)
3. Verify that the user can access the scope

Similar to how it's done here: https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/src/dam/images/images.controller.ts#L97